### DISCUSSION TITLE
Traffic not display on dashboard but can see flow received?

### DISCUSSION THREAD

**Discussion by kronicklez** [akvorado/akvorado#1424] [Q&A] [answered]
Hi vincent / expert,

May i know how to solve issue traffic/graph not display in dashboard even we can see flow received in akvorado. I'm already read troubleshooting section but all give the result. I'm using juniper router and akvorado docker. Below is the snapshot. Appreciate any feedback.

![image](https://github.com/user-attachments/assets/fe96add4-1288-49d5-a006-e9209bc54adc)

**Comment by vincentbernat** [comment, marked as answer]
Likely, this is a classification issue. Go to the visualize tab and remove the filter. You should see your flows.

**Comment by ABEKE8** [comment]
Hi i Vincent, I have also this problem, I removed the filter, but  don't viewi Top source AS,Top source ports,Top protocols,Top source countries,IPv4/IPv6

**Reply by vincentbernat** [reply]
Please, don't spam all the discussions. You need to configure the interface classification for the widgets on the homepage to work.

**Reply by ABEKE8** [reply]
these configurations have been set up:

 interface-classifiers:
    - |
      ClassifyConnectivityRegex(Interface.Description, "^(?i)(transit|peering):? ", "$1") &&
      ClassifyProviderRegex(Interface.Description, "^\\S+?\\s(?:to )?(\\S+)", "$1") &&
      ClassifyExternal()
    - ClassifyConnectivityRegex(Interface.Description, "^(?i)(cust|core):? ", "$1") &&
      ClassifyInternal()
    - ClassifyInternal()

**Comment by vincentbernat** [comment]
If you don't get anything on the homepage, it would mean that you have no flows classified as external. You can check the sample flows also displayed on the homepage to understand where is the problem: are the interface description correctly fetched? do they match the regex?

Also, inside YAML, you need to escape the `\` a second time, so use `\\S` instead of `\S`.

**Reply by ABEKE8** [reply]
Thank you, all work.
